bindings/python/MANIFEST.in

```diff
@@ -1,2 +0,0 @@
-include README.rst
```

Comment:
This file is not needed since we are just using the `package_data` dict for this

Reply:
What does the source dist contain when you do `python setup.py sdist`?